src/apis/apis.type.ts

```diff
@@ -0,0 +1,28 @@
+export type Obj = Record<string, unknown>;
+
+export type StringObj = Record<string, string>;
+
+export type OrderBy = "recent" | "favorite";
+
+export interface GetProductsParams extends Obj {
+  page: number;
+  pageSize: number;
+  orderBy: OrderBy;
+  keyword?: string;
+}
+
+export interface GetProductsRes extends Obj {
+  totalCount: number;
+  list: {
+    createdAt: string;
+    favoriteCount: number;
+    ownerNickname: string;
+    ownerId: number;
+    images: string[];
+    tags: string[];
+    price: number;
+    description: string;
+    name: string;
+    id: number;
+  }[];
```

Comment:
P1:
아래와 같이 타입을 분리하시는 것을 추천드립니다.
이렇게 product 타입을 분리해두면 나중에 사용하시기 더 편리하고 가독성에도 좋을 것 같습니다.

```suggestion
  list: ProductProps[];
```